### PR TITLE
fix: create tx in several SQLs

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/services/tx/transaction-persistor.ts [hunk 1]
```diff
@@ -157,10 +158,44 @@ export class TransactionPersistor {
...
+      await queryRunner.commitTransaction();
+    } catch (err) {
+      await queryRunner.rollbackTransaction()
+    } finally {
+      await queryRunner.release()
+    }
     return tx
   }
 
+  private static waitTransaction = async(queryRunner: QueryRunner, timeout: number = 5000) => {
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/tx/transaction-persistor.ts]: What does this mean? Wait for a transaction to do what?

Comment (classicalliu) [reply packages/neuron-wallet/src/services/tx/transaction-persistor.ts]: Wait transaction end.

Comment (Keith-CY) [reply packages/neuron-wallet/src/services/tx/transaction-persistor.ts]: It seems to be `waitUntilTransactionFinished`

Comment (classicalliu) [reply packages/neuron-wallet/src/services/tx/transaction-persistor.ts]: updated

**Pull request merged**